lib/Page.js

```diff
@@ -136,11 +137,24 @@ function formatSiteNav(renderedSiteNav) {
       const dropdownTitle = $(this).contents().not('ul');
       // Replace the title with the dropdown wrapper
       dropdownTitle.remove();
-      nestedList.wrap('<div class="dropdown-container"></div>');
-      $(this).prepend('<button class="dropdown-btn">'
-        + `${dropdownTitle} `
-        + `${DROPDOWN_BUTTON_ICON_HTML}\n`
-        + '</button>');
+      // Check if dropdown is expanded by default
+      if (dropdownTitle.toString().includes(DROPDOWN_EXPAND_KEYWORD)) {
+        const keywordRegex = new RegExp(DROPDOWN_EXPAND_KEYWORD, 'g');
+        const dropdownTitleNoKeyword = dropdownTitle.toString().replace(keywordRegex, '');
+        const iconSelector = cheerio.load(DROPDOWN_BUTTON_ICON_HTML);
+        const rotatedIconHtml = iconSelector('i').addClass('rotate-icon');
```

Comment:
Seems like you can just compress the above two lines into just one.

```javascript
const rotatedIconHtml = cheerio.load(DROPDOWN_BUTTON_ICON_HTML)('i').addClass('rotate-icon');
```

Reply:
Had to add `cheerio.load( ... , { xmlMode: false })` , else the `.toString()` and `.html()` will produce self closing tags (not supported in HTML5)